app/inputs/inet_input.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Do we need this?

Reply:
raises exception without input files in active admin forms